scripts/qa_crowbarsetup.sh

```diff
@@ -2163,7 +2163,7 @@ function custom_configuration
             # set a custom region name
             proposal_set_value keystone default "['attributes']['keystone']['api']['region']" "'CustomRegion'"
             # speedup the deployment
-            if iscloudver 8plus; then
+            if iscloudver 8M3plus; then
                 proposal_set_value keystone default "['attributes']['keystone']['identity']['password_hash_rounds']" "4"
```

Comment:
ocatacloud still fails with
`key 'password_hash_rounds:' is undefined.\n"`

maybe explicitly test for `&& ! [[ $cloudsource =~ ocata ]]`